tests/foreman/ui/test_host.py

```diff
@@ -275,6 +275,7 @@ def tracer_install_host(rex_contenthost, target_sat):
     yield rex_contenthost
 
 
+@pytest.mark.e2e
```

Comment:
I think there are other tests as well e.g `test_positive_global_registration_end_to_end`  why I could see only one test per module and not more ?

One more example could be `test_positive_crud_module_streams`